### DISCUSSION TITLE
Multiple cursor keybinds modification

### DISCUSSION THREAD

**Discussion by Bykow** [lapce/lapce#2433] [Ideas]
Hello !

In the [docs](https://docs.lapce.dev/features/multiple-cursor) for multiple cursor, it specifies `Ctrl + Alt`. It would be super usefull to be able to change that to `Ctrl + Shift`.

The reason behind that is that I'm using `Alt` as my main swaywm (or i3) modifier. I'm sure I'm not alone in this !

**Comment by MinusGix** [comment]
On the stable version of Lapce those are called 'Insert cursor above'/'Insert cursor above'. You can open keybindings (in settings) and change them to Ctrl + Shift + Up / Ctrl + Shift + Down.

**Reply by Bykow** [reply]
This is not what I'm talking about, I want to change the modifier to interact with the mouse and click lines.

**Reply by MinusGix** [reply]
Ah, I see what you mean now. This is currently hardcoded, apparently. I'd suggest opening an issue for it, so that we can track it.
